ui/components/component-library/avatar-network/avatar-network.js

```diff
@@ -102,17 +102,17 @@ AvatarNetwork.propTypes = {
    * The background color of the AvatarNetwork
```

Comment:
This follows our other conventions of these propTypes. No propType errors in console.